plugins/inputs/knx_listener/README.md

```diff
@@ -0,0 +1,66 @@
+# KNX input plugin
+
+The KNX input plugin that listens for messages on the KNX home-automation bus.
+This plugin connects to the KNX bus via a KNX-IP interface.
+Information about supported KNX message datapoint types can be found at the
+underlying "knx-go" project site (https://github.com/vapourismo/knx-go).
+
+### Configuration
+
+This is a sample config for the plugin.
+
+```toml
+# Listener capable of handling KNX bus messages provided trough a KNX-IP Interface.
+[[inputs.KNXListener]]
+  # Type of KNX-IP interface.
+  # Can be either "tunnel", "router" or "dummy" (for testing only).
+  service_type = "tunnel"
+
+  # Address of the KNX-IP interface.
+  service_address = "localhost:3671"
+
+  ## Measurement definition(s)
+  # [[inputs.KNXListener.measurement]]
+  #   # Name of the measurement
+  #   name = "temperature"
+  #   # Datapoint-Type (DPT) of the KNX messages
+  #   dpt = "9.001"
+  #   # List of Group-Addresses (GAs) assigned to the measurement
+  #   addresses = ["5/5/1"]
+
+  # [[inputs.KNXListener.measurement]]
+  #   name = "illumination"
+  #   dpt = "9.004"
+  #   addresses = ["5/5/3"]
+```
+
+#### Measurement configurations
+
+Each measurement only containx one datapoint-type (DPT) and assigns a list of
+addresses to this measurement. You can, for example group all temperature sensor
+messages within a "temperature" measurement. However, you are free to split
+messages of one datapoint-type to multiple measurements.
+
+**NOTE: You should not assign a group-address (GA) to multiple measurements!**
```

Comment:
Maybe expand as to why this should not be done. Seems like a viable option to track a specific value from a single KNX Telegram in multiple measurements.

Reply:
Currently the plugin cannot handle this case as I construct a map mapping the GA to the data-description. We can add this function later I think. However we should check in the code that GAs are not specified multiple times.